sbc_compassion/model/sponsorship_correspondance.py

```diff
@@ -0,0 +1,109 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014-2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emmanuel Mathier <emmanuel.mathier@gmail.com>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp import fields, models, api, _
+
+
+class ResSponsorshipCorrespondence(models.Model):
+
+    """ This class create correspondances to match Compassion needs.
+    """
+
+    _name = 'sponsorship.correspondence'
+
+    ##########################################################################
+    #                                 FIELDS                                 #
+    ##########################################################################
+
+    sponsorship_id = fields.Many2one(
+        'recurring.contract', 'Sponsorship', required=True)
+    name = fields.Char('name', compute='_set_name')
```

Comment:
`'name'` parameter is not needed, as by default the field name will be equivalent to the field name in the class.